nectar/src/config/settings.rs

```diff
@@ -483,6 +494,10 @@ impl Settings {
                 || Ethereum::default_from_chain_id(comit_network.unwrap_or_default().into()),
                 |file| Ethereum::from_file(file, comit_network),
             )?,
+            sentry: match sentry {
+                None => None,
+                Some(sentry) => Some(Sentry { url: sentry.url }),
+            },
```

Comment:
Same thing here, please extract the actual mapping to a dedicated function of some sorts.